src/components/Button/ButtonStyle.module.scss

```diff
@@ -0,0 +1,24 @@
+@use "../../styles/colors.scss";
+
+.button {
+  background-color: colors.$white;
+  color: colors.$grey;
+  text-align: center;
+  font-size: 14px;
+  font-weight: 400;
+  width: 106px;
```

Comment:
Don't provide fixed width and height to a button. 
Use padding to keep the size dynamic.

Reply:
Implemented